code/controllers/MailChimpController.php

```diff
@@ -79,7 +87,7 @@ public function McSubscribeForm() {
 						new FormAction('McDoSubscribeForm', 'Send Now')
 		);
 		$required = new RequiredFields(
-						array('Email')
+						array('Email', 'lname', 'fname')
```

Comment:
Good Idea adding firstname and lastname, but we can't force pepole to have them always required. They should be configurable, and also "robots" field too.